public/pages/Findings/containers/Findings/Findings.tsx

```diff
@@ -252,7 +264,7 @@ class Findings extends Component<FindingsProps, FindingsState> {
   generateVisualizationSpec() {
     const visData: FindingVisualizationData[] = [];
 
-    this.state.filteredFindings.forEach((finding: FindingItemType) => {
+    this.state.filteredFindings.forEach((finding: any) => {
```

Comment:
Does the `FindingItemType` no longer align with the schema for the findings that are returned? If so, would it be possible to update the `FindingItemType` to accommodate the new structure, or perhaps make a new type? Using type `any` can make understanding the code a little harder.